interactions/api/events/processors/reaction_events.py

```diff
@@ -14,6 +15,8 @@ class ReactionEvents(EventMixinTemplate):
     async def _handle_message_reaction_change(self, event: "RawGatewayEvent", add: bool) -> None:
         if member := event.data.get("member"):
             author = self.cache.place_member_data(event.data.get("guild_id"), member)
+        elif guild_id := event.data.get("guild_id"):
+            author = await self.cache.fetch_member(guild_id, event.data.get("user_id"))
         else:
             author = await self.cache.fetch_user(event.data.get("user_id"))
```

Comment:
You need to account for the edge case where `guild_id` is set, the member is not, and the member is not cached

Reply:
Isn't that already the case? If the member is not cached, then it will fetch it automatically no?

Reply:
I cant even remember what my thought process was 14 hours ago. resolving